text-image-search/src/python/README.md

```diff
@@ -14,28 +26,35 @@ export IMG_DIR=<image-folder>
 ```
 
 ## Compare pre-trained CLIP models for text-image retrieval
-> Create, deploy, feed and evaluate the Vespa app using the Vespa python API
 
-### Jupyter Notebook
+You can follow [this Jupyter
+Notebook](https://github.com/vespa-engine/sample-apps/blob/master/text-image-search/src/python/compare-pre-trained-clip-for-text-image-search.ipynb)
```

Comment:
I think we can do `src/python/compare-pre-trained-clip-for-text-image-search.ipynb`?